### PR TITLE
Added StandedEbooks to meta.yaml

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes Where can I check the output of the CI checks? Sorry I can't seem to find that and figure out what went wrong. Thanks!

**Comment by jakirkham** [comment]
Yeah the UI here could be a bit clearer. Please see screenshots below:

<details>

Screen Shot 2021-03-11 at 9 16 11 AM

Screen Shot 2021-03-11 at 9 16 20 AM

Screen Shot 2021-03-11 at 9 16 27 AM

Screen Shot 2021-03-11 at 9 16 35 AM

Screen Shot 2021-03-11 at 9 16 42 AM

</details>

**Comment by thewchan** [comment]
@jakirkham It looks like a case of unsolvable dependencies? For example `titlecase==0.13.0`? Maybe this version of `titlecase` isn't available on conda-forge? (I do see the package itself on the feedstock https://github.com/conda-forge/titlecase-feedstock)

**Comment by chrisburr** [comment]
>  Maybe this version of titlecase isn't available on conda-forge?

Yup: https://anaconda.org/conda-forge/titlecase/files

If you need it you can make a PR and ask the titlecase-feedstock maintainers using the procedure in https://conda-forge.org/docs/maintainer/updating_pkgs.html?highlight=branches#maintaining-several-versions

**Comment by thewchan** [comment]
Thanks @chrisburr , looks like there are 4 packages missing past versions; I'll spend some time to get those on Conda-forge.

**Comment by thewchan** [comment]
@conda-forge-admin, please rerender

**Comment by thewchan** [comment]
@jakirkham 
@chrisburr 

Finally got everything to pass!

File: recipes/standardebooks/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+about:
+  home: https://standardebooks.org
+  summary: The toolset used to produce Standard Ebooks epub ebooks.
+  dev_url: https://github.com/standardebooks/tools/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: se/data/epubcheck/LICENSE.txt
+  description: |
+    A collection of tools Standard Ebooks uses to produce its ebooks, including basic setup of ebooks, text processing, and build tools.
```
Comment (chrisburr) [inline recipes/standardebooks/meta.yaml:66]: These lines should be wrapped to ~80 characters

**Pull request merged**